doc/level1.rst

```diff
@@ -107,13 +107,26 @@ following attributes with ``peers[addr]``:
 
 * ``last_seen``: The UTC timestamp of the most recent effective date
   (:ref:`definition <effective_date>`) of all messages that the MUA has
-  processed for this peer.
-* ``last_seen_autocrypt``: The UTC timestamp of the most recent effective
-  date of all messages with a valid Autocrypt header that the MUA has
-  processed for this peer.
-* ``public_key``: The public key of the peer.
-* ``state``: A quad-state: ``nopreference``, ``mutual``, ``reset``, or
-  ``gossip``.
+  processed from this peer.
+* ``autocrypt_timestamp``: The UTC timestamp of the most recent
+  effective date of all messages containing a valid ``Autocrypt`` header
+  that the MUA has processed from this peer.
+* ``public_key``: The value of the ``keydata`` attribute derived from
+  the most recent ``Autocrypt`` header received from the peer.
+* ``prefer_encrypt``: The ``prefer-encrypt`` value (either
+  ``nopreference`` or ``mutual``) derived from most recent ``Autocrypt``
+  header received from the peer.
+
```

Comment:
nit: it's not the most recently seen Autocrypt header, but the youngest Autocrypt header that has ever been seen for the given peer.